MAUI/Themes/Keys.md

```diff
@@ -10629,6 +10629,30 @@ This page lists the keys for each control and the element to which it is mapped
             <br/>
         </td>
     </tr>   
+     <tr>
+        <td>
+          SfPdfViewerControlBackgroundColor
```

Comment:
Do we mention as "SfPdfViewer" or "SfPdfViewer control" in all other places?

Reply:
We have mentioned "SfPdfViewer" sir